packages/firestore-compat/src/index.console.ts

```diff
@@ -98,6 +102,15 @@ export class Firestore extends FirestoreCompat {
       new MemoryPersistenceProvider()
     );
   }
+
+  INTERNAL = {
+    delete: () => this.terminate(),
+    count: (query: ExpQuery<unknown>) => {
+      return getCountFromServer(query).then((response: AggregateQuerySnapshot<AggregateSpec>) => {
```

Comment:
Do you really need to provide the type of the `response: AggregateQuerySnapshot<AggregateSpec>` argument? Can't you just omit the type and let the compiler infer it?

Reply:
removed